### ISSUE TITLE
gpg key list is no longer accurate

### ISSUE THREAD

**Issue by ykuksenko** [oVirt/ovirt-site#3114]
The gpg key list shown below is no longer accurate.
https://github.com/oVirt/ovirt-site/blob/eff20922e2b435157fb3f57cfab2140669b156d6/source/download/rpms_and_gpg.md?plain=1#L80-L86
Per https://github.com/oVirt/ovirt-release/issues/298 there is a new key out there. Please list this key. I think the key id is `44E475CB24901D0C`.

**Comment by ykuksenko** [comment]
The gpg instructions are also out of date for AlmaLinux9 (rhel9 based distros?) now.

**Comment by ctodea** [comment]
This issue is also preventing us from installing ovirt-engine-appliance-4.5-20221206125848.1.el9.x86_64.rpm

**Comment by Nihvel** [comment]
I'm having this issue too, this is my temporary fix. 

Edit `/etc/yum.conf` and append:
`exclude=ovirt-engine-appliance-4.5-20221206125848.1.el9`

The playbook (and the user too with dnf/yum) can now install the previous engine appliance version.

Alternatively, one can pre-install the engine appliance with
`dnf install <http-link-to-engine-appliance.rpm>` and let the playbook skip that task.

Detailed steps:
* Do a `dnf install ovirt-engine-appliance`
* Notice the (as of 21st March, latest) version `4.5-20221206125848.1.el9`
* Try to install it. It fails due to GPG key
* Use a different version: `dnf install ovirt-engine-appliance-4.5-20221026100609.1.el9`
* The ovirt engine is installed

**Comment by MaoliWolf** [comment]
With #3278 merged, both halves of this should now be addressed:

  - The new `24901D0C` key was added to the *Currently used keys* table in [`9163c1e2`](https://github.com/oVirt/ovirt-site/commit/9163c1e212) (2024-02-26), but the commit wasn't linked to this issue, which is likely why it stayed open.
  - The *Importing Keys Manually* section, which still pointed at the legacy `FE590CB7`, now uses the current oVirt 4.5 key via `centos-release-ovirt45` (#3278).

  Looks closable to me — @dupondje, any objection?

**Comment by dupondje** [comment]
Agree. To make it close automatically, you need 'Fixes #id' in the commit msg :)
